src/__tests__/sdk_codes.test.ts

```diff
@@ -0,0 +1,53 @@
+import { Logger } from '../log';
+import * as sdkCodes from '../sdk_codes';
+
+describe('SDK Tests', () => {
```

Comment:
SDK Tests is extremely vague. What are we actually testing here? What is the "Unit" we're exercising? The word "Tests" is superfluous since we know these are tests. Use just the name of the unit.

Reply:
Renamed to `SDK Codes`